RPLCD/i2c.py

```diff
@@ -77,7 +110,16 @@ def __init__(self, address, port=1,
 
     def _init_connection(self):
         self.bus = SMBus(self.port)
-        c.msleep(50)
+
+        if(self.i2c_expander == "PCF8574"):
+            c.msleep(50)
+        elif(self.i2c_expander == "MCP23008"):
+            # Set IO DIRection to output on all GPIOs (GP0-GP7)
+            self.bus.write_byte_data(self.address, c.MCP23008_IODIR, 0x00)
```

Comment:
Is `0x00` the constant for "output"? If yes, maybe put it into a variable called `MCP23008_IODIR_OUT` or `MCP23008_MODE_OUT` or something like that?